webapp/channels/src/components/menu/menu.tsx

```diff
@@ -303,7 +310,7 @@ function MenuModal(props: MenuModalProps) {
         <CompassDesignProvider theme={theme}>
             <GenericModal
                 id={props.menuId}
-                className='menuModal'
+                className='menuModal LOOOOOOOOOOOOOOOOOOOOOOOOOOOOOL'
```

Comment:
😆

Reply:
Sometimes the most trivial methods of debugging are the most effective 😀

Reply:
Removed.